Packs/Druva/Integrations/Druva/Druva.py

```diff
@@ -37,12 +35,36 @@ def get_quarantineRanges(self):
 
     def get_findDevice(self, search_string):
         params = {'hostname': search_string}
-        return self._http_request(method='GET', url_suffix='/realize/ransomwarerecovery/v1/search/resource',
+        return self._http_request(method='GET', url_suffix='/realize/ransomwarerecovery/v1/search/backupset',
                                   params=params, resp_type='response')
 
-    def post_quarantineResource(self, resource_id, resource_type, from_date, to_date):
-        json_data = {'resourceType': resource_type, 'fromDate': from_date, 'toDate': to_date}
-        url_suffix = '/realize/ransomwarerecovery/v1/quarantineranges/resource/' + resource_id
+    def get_findUser(self, user_string):
+        params = {'users': user_string}
+        return self._http_request(method='GET', url_suffix='realize/ransomwarerecovery/v1/users',
+                                  params=params, resp_type='response')
+
+    def get_findUserDevice(self,userId):
+        params = {'users[]':userId,'resourceTypes[]':['Endpoint','OneDrive','Google Drive']}
+        return self._http_request(method='GET', url_suffix='realize/ransomwarerecovery/v1/search/device',
+                                  params=params, resp_type='response')
+
+    def get_findSharePointSites(self,search_string):
+        params = {'siteTitlePrefix': search_string}
+        return self._http_request(method='GET', url_suffix='/realize/ransomwarerecovery/v1/search/sharepoint-sites',
+                                  params=params, resp_type='response')
+
+    def get_findSharedDrives(self,search_string):
+        params = {'accountTitlePrefix': search_string}
+        return self._http_request(method='GET', url_suffix='/realize/ransomwarerecovery/v1/search/shareddrive-accounts',
+                                  params=params, resp_type='response')
+
+
+    def post_quarantineResource(self,org_id,resource_id, resource_type, from_date, to_date):
+        if org_id is not None:
```

Comment:
Are we expecting `org_id` to be defined? What happens if `org_id` is `None`? Currently, we still send it in `json_data`.